packages/jest-environment/src/index.ts

```diff
@@ -148,6 +149,11 @@ export interface Jest {
    * Not available when using legacy fake timers implementation.
    */
   getRealSystemTime(): number;
+  /**
+   * Fetches the seed generated for this particular running of Jest.
+   * It can also be manually set with --seed CLI argument.
+   */
+  getSeed(): number;
```

Comment:
Please add a basic type test to this file:

https://github.com/facebook/jest/blob/fd13a104ea8fbd5b758cb4e09a58ceb41fd7b822/packages/jest-types/__typetests__/jest.test.ts#L95-L99

Something like:

```ts
expectType<number>(jest.getSeed()); 
expectError(jest.getSeed(123)); 
```

To run the type tests – build the library and run `yarn test-types`.